### ISSUE TITLE
Some warning logs should be meta-linting issues

### ISSUE THREAD

**Issue by stdedos** [golangci/golangci-lint#3403]
### Your feature request related to a problem? Please describe.

This warning

```
level=warning msg="[runner/nolint] Found unknown linters in //nolint directives: 0 or 1 lenght slices are already sorted"
```

and maybe that warning too

```
level=warning msg="[linters_context] wastedassign is disabled because of generics. You can track the evolution of the generics support by following the https://github.com/golangci/golangci-lint/issues/2649."
```

Should be instead meta-issues, at least for the purposes of `format: code-climate` output.

### Describe the solution you'd like.

For `format: code-climate` output, in order not to loose "syntax issues like these", they should be included in the `format: code-climate` output.

### Describe alternatives you've considered.

In this case, for the `//nolint`, the `nolintlint` would've helped, I guess
